code/game/objects/structures/crates_lockers/crates.dm

```diff
@@ -45,27 +46,28 @@
 
 /obj/structure/closet/crate/attack_hand(mob/user)
 	. = ..()
-	if(.)
-		return
 	if(manifest)
-		tear_manifest(user)
+		tear_manifest(FALSE, FALSE, user)
 
 /obj/structure/closet/crate/open(mob/living/user)
 	. = ..()
-	if(. && manifest)
-		to_chat(user, "<span class='notice'>The manifest is torn off [src].</span>")
-		playsound(src, 'sound/items/poster_ripped.ogg', 75, TRUE)
-		manifest.forceMove(get_turf(src))
-		manifest = null
-		update_icon()
-
-/obj/structure/closet/crate/proc/tear_manifest(mob/user)
+	if(manifest)
+		tear_manifest(FALSE, FALSE, user)
+
+/obj/structure/closet/crate/proc/tear_manifest(datum/source, obj/item/S, mob/user)

```

Comment:
shouldnt you flip this and do it on the folder/clipboard component's preattack?  
register to parent's `COMSIG_ITEM_PRE_ATTACK` then tear the manifest and return `COMPONENT_NO_ATTACK` if its something with a manifest?  or even add a manifest component?

Reply:
what